modules/test/playwright/fixtures/objectPagesTest.ts

```diff
@@ -5,29 +5,54 @@
 
 import {test} from '@playwright/test';
 
+import {EditObjectDefinitionPage} from '../pages/object-web/EditObjectDefinitionPage';
 import {ModalAddObjectDefinitionPage} from '../pages/object-web/ModalAddObjectDefinitionPage';
 import {ModalEditObjectFolderPage} from '../pages/object-web/ModalEditObjectFolderPage';
 import {ModelBuilderPage} from '../pages/object-web/ModelBuilderPage';
 import {ViewObjectDefinitionsPage} from '../pages/object-web/ViewObjectDefinitionsPage';
+import {ViewObjectEntriesPage} from '../pages/object-web/object-entries/ViewObjectEntriesPage';
+import {EditObjectValidationPage} from '../pages/object-web/object-validation/EditObjectValidationPage';
+import {ModalAddObjectValidationPage} from '../pages/object-web/object-validation/ModalAddObjectValidationPage';
+import {ObjectValidationsFDSPage} from '../pages/object-web/object-validation/ObjectValidationsFDSPage';
 
 const objectPagesTest = test.extend<{
+	editObjectDefinitionPage: EditObjectDefinitionPage;
+	editObjectValidationPage: EditObjectValidationPage;
 	modalAddObjectDefinitionPage: ModalAddObjectDefinitionPage;
+	modalAddObjectValidationPage: ModalAddObjectValidationPage;
 	modalEditObjectFolderPage: ModalEditObjectFolderPage;
 	modelBuilderPage: ModelBuilderPage;
+	objectValidationsFDSPage: ObjectValidationsFDSPage;
```

Comment:
Hey @igor-franca, for me, just `objectValidationsPage` doesn't give the idea that we are in the FDS table. In the case of object validations, that's fine because we only have them in this location, but if for some reason we create a new object validation listing in another location, we will have to put more specific names, I don't know this change will be worth it because of the probability of generating more work in furure because from the generic name.